conda_pack/core.py

```diff
@@ -874,78 +865,80 @@ def __init__(self, prefix, archive, dest_prefix=None):
         self.prefixes = []
 
     def add(self, file):
+        # Windows note:
+        # When adding files to an archive, that archive is generally
+        # case-sensitive.  The target paths can be mixed case, and that means
+        # that they will be distinct directories in the archive.
+        #
+        # If those files are then extracted onto a case-sensitive file-system
+        # (such as a network share), Windows will not be able to traverse them
+        # correctly.
```

Comment:
To be clear, we're proposing to punt on this hypothetical. We can gather more information on this if we can build a good test case (e.g., an environment saved on a case-sensitive filesystem).

Reply:
I don't understand what this hypothetical means. The solution here seems fine, but it's not clear to me why it's even needed.

Reply:
@dsludwig you may be able to explain this better than I. Did you have a specific scenario where you thought this would happen? I think IIRC you actually had some issues when testing on a case-sensitive filesystem.

Reply:
Yeah! My specific scenario happened when I unpacked an environment containing R, although I think there may have been problems with Python as well.

The `info/files.json`/paths for some packages starts with `Lib\`, and for other packages starts with `lib\`. Normally, if you extract that on Windows, they would be unioned together. However, if you extract that on a case-sensitive FS, they are distinct.

I was testing this serving an ext3 file-system over SMB.

Reply:
OK, I remember now. The issue is that Anaconda R packages are `noarch: generic`, and unpack into `lib/`.

Reply:
That's right. This is because we want to start packaging R packages as `noarch: generic`. We need the library directory to be lowercase `lib` for Unix. In theory, if the _only_ time we're using `lib` on Windows is these R packages, we can be reasonably sure it will work just fine; R will live in `lib`, Python in `Lib`.